config/config.yml

```diff
@@ -1,19 +1,26 @@
 app: 'sigma_hunting_app'
 cron_schedule: '*/10 * * * *'
-earliest_time: '-10m'
+earliest_time: '0'
```

Comment:
Shouldn't you search for the last 10m when you have a cron schedule of every 10m?

Reply:
In fact all my searches use indexed time to prevent missing alert if delay are observed on collect